server/cars.js

```diff
@@ -29,6 +32,44 @@ const carSchema = new mongoose.Schema({
 
 const Car = mongoose.model('Car', carSchema);
 
+
+//--------HELPER FUNCIONS--------//
+/**
+ * Returns the mileage the service is next due based on 
+ * when it was last done and the interval
+ * @param {number} currentMileage - the current mileage of the car
+ * @param {number} interval - how often the service is done (miles)
+ * @param {number} lastDone - the mileage the service was last done
+ */
+const getNextService = (currentMileage, interval, lastDone) => {
```

Comment:
I condensed the functions you had for each of the services into the one. If the last done isn't provided it'll follow the algo you had before, but if it is provided it'll just add the interval on to the last done